### PR TITLE
DB-3180: Page Grid Page

### PR DESCRIPTION
**Pull request by enZane**
## What changes were made?
- Added the file Pages.js to the lib, that has the function to query the latest pages
- Added the PageGridItem component to render a page
- Added the PagesGrid to the /pages route

## Where were the changes made?
Next-Wordpress-Starter

## How have the changes been tested?
Tested locally and build, also I created a Page with a featured image in the wordpress to test that behavior, after that I deleted it, if is necessary I can keep it

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: starters/next-wordpress-starter/components/grid.js [hunk 1]
```diff
@@ -58,3 +58,37 @@ export const PostGridItem = ({ content: post }) => {
     </Link>
   );
 };
+
+export const PageGridItem = ({ content: page }) => {
+  const imgSrc = page?.featuredImage?.node.sourceUrl || "";
+  const altText = page?.featuredImage?.node.altText || page.title;
+
+  return (
+    <Link passHref href={`/page${page.uri}`}>
```
Comment (CobyPear) [inline starters/next-wordpress-starter/components/grid.js:67]: I think we'll want the prefix here to be `/pages`

**Review by CobyPear — CHANGES_REQUESTED** [review]
Just needs a changeset and one tiny change to the url prefix.

File: starters/next-wordpress-starter/components/grid.js (hunk 1 above)
Comment (enZane) [reply starters/next-wordpress-starter/components/grid.js]: Nice catch!

**Review by CobyPear — APPROVED** [review]
Looks good!

**Pull request merged**